ldfparser/ldf.lark

```diff
@@ -53,7 +53,7 @@ diagnostic_frames: "Diagnostic_frames" "{" (diagnostic_frame_definition)+ "}"
 diagnostic_frame_definition: ldf_identifier ":" ldf_integer "{" (ANY_SEMICOLON_TERMINATED_LINE)* "}"
 
 // LIN 2.1 Specification, section 9.2.2.2
-node_attributes: "Node_attributes" "{" (node_definition+) "}"
+node_attributes: "Node_attributes" "{" (node_definition*) "}"
```

Comment:
In the specification it requires at least one node attribute, just like in the Nodes section it requires at least one slave. I'll later verify whether this is allowed using the Vector LDF Explorer.